### ISSUE TITLE
WARN: unable to check pg-1: [ProtocolError] expected value '2.23' for greeting key 'version' but got '2.26'

### ISSUE THREAD

**Issue by MichaelDBA** [pgbackrest/pgbackrest#1026]
WARN: unable to check pg-1: [ProtocolError] expected value '2.23' for greeting key 'version' but got '2.26'

OK so I upgraded pgbackrest from 2.23 to 2.26 on all my pg 12.2 hosts on CentOS 7.7 including remote backup host where pgbackrest repo resides.

I was unable to delete the stanzas after I upgraded, so I deleted everything under the pgbackrest archive and backup directories, which worked in the past.  But still the same error comes up.  Where is pgbackrest getting a reference to the old version, 2.23?

Installed prior and current versions of pgbackrest via packages.

So, basically, I want to understand how the heck pgbackrest even knows about me using 2.23 previously, since I deleted all references to my specific stanza in the pgbackrest repo.

**Comment by MichaelDBA** [comment]
Sorry, I was clumsy.  Some hosts were still at 2.23 version
